editor/property_selector.cpp

```diff
@@ -31,10 +31,16 @@
 #include "property_selector.h"
 
 #include "editor_scale.h"
+#include "modules/visual_script/visual_script.h"
```

Comment:
`editor/` files can't be made to depend on a module, as said module could be disabled and compilation will fail. If you need to make VisualScript editor features, they should go in `modules/visual_script/`.

Reply:
Resolved this issue by moving property selector to modules/visual_script/.